File: apps/service-with-helm/src/APIs/Order/Dtos/Order.cs
namespace ServiceWithHelm.APIs.Dtos;

public class Order
{
    public DateTime CreatedAt { get; set; }

    public string? Customer { get; set; }

    public string? Details { get; set; }

    public string Id { get; set; }

    public double? Price { get; set; }

    public DateTime UpdatedAt { get; set; }
}

File: apps/import-test-2-admin/src/api/order/Order.ts
import { Customer } from "../customer/Customer";
import { Product } from "../product/Product";

export type Order = {
  createdAt: Date;
  customer?: Customer | null;
  discount: number | null;
  id: string;
  product?: Product | null;
  quantity: number | null;
  status?: "PENDING" | "COMPLETED" | "CANCELLED" | null;
  totalPrice: number | null;
  updatedAt: Date;
};
